.github/workflows/publish.yml

```diff
@@ -40,8 +40,19 @@ jobs:
       - name: Install mdbook
         uses: ./.github/workflows/install-mdbook
 
+      - name: Install mdbook-pandoc
+        run: cargo install mdbook-pandoc
```

Comment:
Small thing: I started adding ` --locked --version x.y.z` to the `cargo install` lines after we ran into incompatibilities between a new version of `mdbook` and one of the plugins we use. I would suggest doing the same here for consistency.